extending-dummy4j.md

```diff
@@ -41,20 +41,29 @@ If you choose to, you can place all of your definitions in one file. You can als
 
 ## Parser
 
-The parser recognizes two placeholders:
+The parser recognizes three placeholders:
 * `#{key.path}` - will resolve to a random value from the list of data definitions using the provided key path; The
```

Comment:
In `ExpressionResolver` docs we use `#{definition.path}}`. Is this the same as `#{key.path}`?

Furthermore, I don't know if I understand the `#{key.path}` thing correctly. Shouldn't it be `#{path.key}`?

Reply:
`key.path` is meant to be "key path". Inversed, it would have to be `path.to.key`. It's the same as `definition.path`. Might be a good idea to unify this.